### ISSUE TITLE
Navigation timeout of default 30000ms when setting 0 for infinite

### ISSUE THREAD

**Issue by landonairey** [jef/nvidia-snatcher#401]
After running the tool there is an error for navigation timeout:
"[nvidia] nvidia founders edition - Navigation timeout of 30000 ms exceeded"

However, setting the PAGE_TIMEOUT variable to 0 should let this run beyond 30s according to the description

Unique fields set are:
IN_STOCK_WAIT_TIME="1"
PAGE_TIMEOUT="0"
SHOW_ONLY_MODELS="founders edition"
SHOW_ONLY_SERIES="3080"

The variable possibly isn't being set

**Comment by jef** [comment]
You're right. This is a bug. I'm looking at alternative functions to ensure this timeout gets used.

**Comment by landonairey** [comment]
Thanks for the quick reply. I wanted to sort out if this was operator error or not so I tested with setting just these parameters:
<SHOW_ONLY_SERIES="3080">
<STORES="newegg">

However, when running the tool, the first couple of print outs show that the default values are being used:
<[2:05:35 PM] info :: [] selected stores: nvidia>
<[2:05:35 PM] info :: [] selected series: 3070, 3080, 3090>

Maybe the .env file is not being read at all. I modified the .env-example file with values and saved it in the same directory as "fe3080.env". Did I miss any steps?
